Addressing Eric V's adn Erik K's telechat reviews (#33)
martinduke/congestion-control-charter: congestion-control-charter.txt

﻿CONGestion RESponse and Signaling Working Group Charter (CONGRESS)

RFC 5033 describes a Best Current
Practice to evaluate new congestion control algorithms as Experimental or
Proposed Standard RFCs. TCP was the dominant

> consumer of this work, and proposals typically discussed in research groups (for example, Internet Congestion Control Research Group - ICCRG).

Reviewer: ```suggestion
consumer of this work, and proposals were typically discussed in research groups (for example, Internet Congestion Control Research Group - ICCRG).

```

make it a natural venue to take on other work related to indications of
congestion, such as delay, and potentially queueing algorithms.

> CONGRESS is chartered to conduct a revisionof RFC 5033. The working group should

Reviewer: ```suggestion
CONGRESS is chartered to conduct a revision of RFC 5033. The working group should

```

> CONGRESS is chartered to conduct a revisionof RFC 5033. The working group should

Reviewer 2: s/revisionof/revision of/

consider a revision that relaxes requirements to encourage more experiments in

> the IRTF/IETF. Adoption of standards-track congestion control algorithm should consider (1) empirical

Reviewer: ```suggestion
the IRTF/IETF. Adoption of standards-track congestion control algorithms should consider (1) empirical

```

> evidence of safety (for example - avoiding congestion collapse) and (2) stated intent to deploy by major transport

Reviewer 2: Still unsure whether "safety" is the right word, but, at least, the example given is useful.

Author: I am afraid we might not have a better alternative word here :-(

implementations.

> CONGRESS is authorized to adopt works relating to Congestion Control and AQM without rechartering.

Reviewer: ```suggestion
CONGRESS is authorized to adopt work relating to Congestion Control and AQM without rechartering.

```

existing standards. Other venues, e.g., IRTF, may be more appropriate for
publishing such documents.

> CONGRESS will not remain open simply “in case” further work comes along. 

Reviewer 2: It addresses my DISCUSS albeit the sentence looks a little weird like it is.

Author: Thanks.
